airflow/providers/openlineage/utils/utils.py

```diff
@@ -307,6 +308,17 @@ class TaskGroupInfo(InfoJsonEncodable):
     ]
 
 
+def get_airflow_dag_run_facet(dag_run: DagRun) -> dict[str, BaseFacet]:
+    if not dag_run.dag:
+        return {}
+    return {
+        "airflow_dagRun": AirflowDagRunFacet(
```

Comment:
Interesting, i missed that part of specification about naming when I created the `airflowState` facet.

Reply:
It is mentioned only once, in OpenLineage.md file, but in the docs on site

Reply:
@mobuchowski WDYT? Should we start change current one and start sending duplicate under `airflow_state` key or change the spec and adjust this new one to `airflowDagRun`?

Reply:
The way to properly do renaming - assuming we want to - is to publish the facet under both names to let consumers switch to the new facet name.

Reply:
Personally I'd stick with current name, even though it's not compliant with the doc. Maybe we should adjust it, but sending duplicates for a long time for newly created facet just because of this mistakes seems like on overkill for me. I think in more and more places we are going with camelCase instead (e.x. [BQ mixin](https://github.com/apache/airflow/blob/42a96412b4cbe309c5c9f0f298a5514de838e9d7/airflow/providers/google/cloud/openlineage/mixins.py#L127))

Reply:
I don't have a strong opinion here, i think I simply want to avoid introducing multiple naming cases that the consumers would have to handle 😉

Reply:
Let's use `airflowDagRun` as a key for now, and discuss this in https://github.com/OpenLineage/OpenLineage/issues/2866.